client/app/scripts/superdesk-desks/views/aggregate-settings.html

```diff
@@ -1,72 +1,68 @@
-<div sd-modal data-model="modalActive" class="add-group-popup modal-large modal-static" >
+<div class="modal-header">
+    <a href="" class="close" ng-click="cancel()"><i class="icon-close-small"></i></a>
+    <h3 translate>Aggregate settings</h3>
+</div>
 
-        <div class="modal-header">
-            <a href="" class="close" ng-click="cancel()"><i class="icon-close-small"></i></a>
-            <h3 translate>Aggregate settings</h3>
-        </div>
+<div class="modal-body" sd-wizard data-name="aggregatesettings" data-current-step="step.current" data-finish="cancel()">
 
-        <div class="modal-body" sd-wizard data-name="aggregatesettings" data-current-step="step.current" data-finish="cancel()">
+    <div sd-wizard-step data-title="{{ :: 'Desks' | translate }}" data-code="desks">
+        <div class="content">
 
-            <div sd-wizard-step data-title="{{ 'Desks' | translate }}" data-code="desks">
-                    <div class="content">
+            <div class="legend" translate>Select desks for view</div>
+            <div class="desk" ng-repeat="desk in desks">
+                <div class="desk-title">
+                    {{desk.name}}
+                    <span sd-switch class="pull-right" ng-model="editGroups[desk._id].selected" ng-click="setDeskInfo(desk._id)"></span>
+                </div>
+                <ul class="stages clearfix" ng-if="editGroups[desk._id].selected">
+                    <li ng-repeat="stage in deskStages[desk._id]">
```

Comment:
Can stages be added to a desk while this view is active? If yes, we should add `track by` here, if not, ignore this comment. :)